loadpipeofficial/Hamza-medusa
# HAM-1863: Shipping cost 0.00 on Thankyou page

On checkout finalization, assigned the cart's shipping method to the newly created order.

https://www.notion.so/hamza-market-token/Shipping-total-doesn-t-show-on-order-complete-template-c1b126ccfd2e4774b3302ecd253ae6f5?pvs=23

To test: 
- create a cart 
- go to checkout 
- note that the shipping cost is > 0
- complete checkout 
- view thankyou page 
- shipping cost displayed should be > 0 (should be the same as shown in step 3)
